app/src/main/java/com/cook/easypan/easypan/data/database/FirestoreClient.kt

```
@@ -76,12 +103,89 @@ class FirestoreClient {
+    ): Boolean {
+        val docRef = firestore.collection(USER_DATA_COLLECTION)
+            .document(userId)
+            .collection(FAVORITE_COLLECTION)
+            .document(recipeId)
+
+        return try {
+                docRef
+                    .delete()
+                    .await()
+                true
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)